CRM/Oapproviderlistapp/Form/Individual.php

```diff
@@ -116,7 +116,7 @@ public function formRule($fields, $files, $self) {
     }
     if (!empty($fields['email'][1]) && !empty($self->_contactID)) {
       // Check to see if email exists as provider.
-      $isProvider = CRM_Core_DAO::singleValueQuery('SELECT IF(is_deleted=0, "Yes", "No") FROM civicrm_email e INNER JOIN civicrm_contact c ON c.id = e.contact_id WHERE email LIKE %1 AND is_deleted <> 1', [1 => [$fields['email'][1], 'String']]);
+      $isProvider = CRM_Core_DAO::singleValueQuery('SELECT IF(is_deleted=0, "Yes", "No") FROM civicrm_email e INNER JOIN civicrm_contact c ON c.id = e.contact_id WHERE e.email LIKE %1 AND c.is_deleted <> 1 AND c.contact_usb_type LIKE \'%Provider%\'', [1 => [$fields['email'][1], 'String']]);
```

Comment:
@seamuslee001 there is a typo in WHERE clause it should be ```c.contact_sub_type``` not ```c.contact_usb_type```

Reply:
Fixed @monishdeb